internal/controllers/core/tiltfile/filewatch.go

```diff
@@ -99,18 +97,30 @@ func ToFileWatchObjects(watchInputs WatchInputs) typedObjectSet {
 		}
 	}
 
+	paths := []string{}
 	if len(watchInputs.ConfigFiles) > 0 {
+		paths = append(paths, watchInputs.ConfigFiles...)
+	} else if watchInputs.TiltfilePath != "" {
+		// A complete ConfigFiles set should include the Tiltfile. If it doesn't,
+		// add it to the watch list now.
+		paths = append(paths, watchInputs.TiltfilePath)
```

Comment:
Given the comment, should we use a set and/or check to see if slice already contains `watchInputs.TiltfilePath`?

Reply:
mayyyyybe? tbqh, I'm not totally convinced we should even auto-reload the tiltfile anymore, and mainly trying to leave it alone right now.